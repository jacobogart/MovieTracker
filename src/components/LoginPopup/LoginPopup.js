import React, { Component } from 'react';
import { LoginForm } from '../../containers/LoginForm/LoginForm';

class LoginPopup extends Component {
  render() {
    return(
      <div className="LoginPopup">
        <h3>Please login</h3>
        <p>You must be logged in to view your favorites</p>
        <LoginForm />
      </div>
    );
  }
}

export default LoginPopup;