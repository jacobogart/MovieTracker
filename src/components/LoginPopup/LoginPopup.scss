.LoginPopup {
  margin: 60px auto 100px;
  width: 600px;

  h3,
  p {
    color: white;
  }

  h3 {
    font-size: 1.6em;
    margin: 0 0 5px 0;
  }

  p {
    font-size: 1.1em;
  }

  .error-message {
    color: $accent;
    font-size: 1.1em;
  }
}