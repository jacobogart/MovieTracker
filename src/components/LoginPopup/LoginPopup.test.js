import LoginPopup from "./LoginPopup";
import { shallow } from "enzyme";
import React from "react";

describe("LoginPopup", () => {
  it("should match the snapshot", () => {
    let wrapper = shallow(<LoginPopup />);
    expect(wrapper).toMatchSnapshot();
  });
});
