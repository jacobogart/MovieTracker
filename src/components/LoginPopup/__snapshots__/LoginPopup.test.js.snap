// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginPopup should match the snapshot 1`] = `
<div
  className="LoginPopup"
>
  <h3>
    Please login
  </h3>
  <p>
    You must be logged in to view your favorites
  </p>
  <ConnectFunction />
</div>
`;
